README.md

```diff
@@ -1,138 +1,72 @@
 
-To run the full stack using [Docker](https://www.docker.com/) (recommended), please use the following instructions:
 
-1. Run all Docker services with `docker compose up --scale backend=4 --build`. This will start the services with four
-   instances of the backend module running in parallel. Note the `--build` flag used to build the images first. If using
-   default images, that flag can be removed. The images can also be built with `docker compose build`.
+<!-- END BADGIE TIME -->
 
-2. The application will be running and available at `http://localhost`.
+*For the previous version of Clowder, please see [Clowder v1](https://github.com/clowder-framework/clowder).*
 
-3. To access the Traefik dashboard, go to `http://localhost:8080`. To view the raw
-   settings, go to `http://localhost:8080/api/rawdata`.
+Clowder v2 is a reimagining of the [Clowder research data management system](https://clowderframework.org/)
+using a different and newer technology stack. Clowder is a cloud native data management framework to support any 
+research domain. Clowder was developed to help researchers and scientists in data intensive domains manage raw data, 
+complex metadata, and automatic data pipelines.
 
-## Developing
+While the Clowder v1 has worked well over the years, many of the underlying
+technologies and libraries have not received enough support in recent years and new developers have had a challenging
+time learning how to contribute to it.
+Clowder v2 is also an opportunity to leverage our experience working with
+research data in Clowder and deliver a better solution to common problems researchers encounter when working with data.
 
-When developing, the required services can be run using Docker. You can then run the backend
-and frontend modules from the command line or in your favorite IDE (to make debugging easier). We recommend
-using [PyCharm](https://www.jetbrains.com/pycharm/) and have
-made our run configurations available in the `.run` folder. PyCharm should automatically import it, but you will have
-to change the path to the Python virtual environment to point to your path on your host (see Initial Dependencies
-section below).
+Clowder v2 provides:
 
-### Initial Development Dependencies
+- a better user experience and user interface
+- an easier code base to pick up and modify written in Python and React
+- new features based on our experience working with researchers
 
-- Run `python3 -m venv venv` to create a Python Virtual Environment and add it to PyCharm by navigating to
-  `PyCharm -> Settings... -> Project: clowder2 -> Python Interpreter -> Add Interpreter`.
-- Run `source venv/bin/activate && pip install --upgrade pip` to activate the created Python Virtual Environment and
-  upgrade
-  pip.
-- Run `pip install pipenv` to install [Pipenv](https://pipenv.pypa.io/en/latest/).
+## Documentation
 
-### Required Services
+The v2 documentation is still work in progress. It's available at https://clowder2.readthedocs.io.
```

Comment:
Just curious, how did you deploy the docs?

Reply:
You register the git repository in readthedocs and it builds it automatically when there is new commits. No actions required.